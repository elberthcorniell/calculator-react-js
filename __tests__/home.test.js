import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Home from '../src/Home';

describe('test', () => {
  test('Render as spected', () => {
    render(<Home />);
    expect(document.body.innerHTML).toMatchSnapshot();
  });

  test('Loads and displays home', () => {
    render(<Home />);
    expect(screen.getByRole('heading')).toHaveTextContent('Home');
  });
});
