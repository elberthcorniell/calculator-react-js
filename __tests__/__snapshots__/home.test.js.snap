// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Render as spected 1`] = `"<div><h1>Home</h1><p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dolores illo error excepturi velit vero sunt ex, voluptate placeat qui pariatur minima quidem nemo est quia fugit sed laborum recusandae esse similique nostrum assumenda. Deserunt distinctio quod, delectus aspernatur voluptas rerum. Nesciunt unde maiores iste expedita magni non necessitatibus consectetur accusantium, sunt iure beatae incidunt obcaecati. Non cumque iure voluptatem iste id? Nam aperiam hic alias doloribus est. Omnis, cupiditate temporibus?</p></div>"`;
